src/coreclr/src/debug/daccess/daccess.cpp

```diff
@@ -72,11 +72,11 @@ BOOL WINAPI DllMain(HANDLE instance, DWORD reason, LPVOID reserved)
         {
             return FALSE;
         }
-#endif
-        InitializeCriticalSection(&g_dacCritSec);
-
+#else
         // Save the module handle.
         g_thisModule = (HINSTANCE)instance;
```

Comment:
Can `g_thisModule` be deleted now? (Or if it needs to stay - ifdef it out for non-Windows at the top of the file?)

Reply:
Yes, of course. This assignment is the only use. 
I thought I removed them all. This one does not have "h" in the name could be missed when I searched for what remains.